raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -1137,9 +1178,9 @@ contract TokenNetwork is Utils, Controllable {
     )
         internal
     {
-        require(channel_identifier == getChannelIdentifier(participant, partner));
-        require(total_deposit > 0);
-        require(total_deposit <= channel_participant_deposit_limit);
+        require(channel_identifier == getChannelIdentifier(participant, partner), "TN/deposit: wrong channel id");
+        require(total_deposit > 0, "TN/deposit: total_deposit is zero");
+        require(total_deposit <= channel_participant_deposit_limit, "TN/deposit: participant deposit limit reached");
```

Comment:
Maybe just "deposit limit reached"? Participant is a long word, and it's anyway clear what we're talking about here since we know we are in `deposit`.

Reply:
👍